samples/Chat/tsconfig.json

```diff
@@ -9,6 +9,7 @@
       "@internal/acs-ui-common": ["../../../packages/acs-ui-common/src"],
       "@internal/calling-stateful-client": ["../../../packages/calling-stateful-client/src"],
       "@internal/chat-stateful-client": ["../../../packages/chat-stateful-client/src"],
+      "@internal/northstar-wrapper": ["../../../packages/northstar-wrapper/src"],
```

Comment:
Good question! We avoid imports in the code to make sure we only use publicly available exports so that when we migrate the code to the samples nothing breaks. But internally it needs to reference the packlets for quicker building (doesn't have to first build the downstream packlets) and for hot reloading